app/views/batch_imports/new.html.erb

```diff
@@ -1,6 +1,6 @@
 <% provide :page_title, curation_concern_page_title(curation_concern) %>
 <% provide :page_header do %>
-  <h1>CSV Import</h1>
+  <h1>Batch Imports</h1>
```

Comment:
@mcritchlow It looks like controller `batch_import` is use by other purpose:
 `             email_batch_import GET|POST /batch_import/:id/email(.:format)                              batch_import#email
                  sms_batch_import GET|POST /batch_import/:id/sms(.:format)                                batch_import#sms
             citation_batch_import GET      /batch_import/:id/citation(.:format)                           batch_import#citation
          email_batch_import_index GET|POST /batch_import/email(.:format)                                  batch_import#email
            sms_batch_import_index GET|POST /batch_import/sms(.:format)                                    batch_import#sms
       citation_batch_import_index GET      /batch_import/citation(.:format)                               batch_import#citation
                batch_import_index POST     /batch_import(.:format)                                        batch_import#create
                  new_batch_import GET      /batch_import/new(.:format)
`
Maybe we need to use other still use terms or `batch_imports` for the controller, while `Batch Import` as text display in the UI?